bindgen-tests/tests/expectations/struct_with_anon_struct_array_float.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
Huh?

Reply:
this seems to be a merge conflict casualty

Reply:
It is not :upside_down_face: the file is just there since 2017: https://github.com/rust-lang/rust-bindgen/blob/master/tests/expectations/struct_with_anon_struct_array_float.rs

Reply:
Apparently the file was created here and it was empty from the start. I'll remove it in an upcoming: https://github.com/fitzgen/rust-bindgen/commit/b4895d96698504dc28dfa64c4c4e4c3f3714a3c8